Apps/Common/test/unit/Services/BroadcastServiceTests.cs

```diff
@@ -0,0 +1,312 @@
+//-------------------------------------------------------------------------
+// Copyright © 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+namespace HealthGateway.CommonTests.Services
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net;
+    using System.Net.Http;
+    using HealthGateway.Common.Api;
+    using HealthGateway.Common.Data.Constants;
+    using HealthGateway.Common.Data.Models;
+    using HealthGateway.Common.Data.ViewModels;
+    using HealthGateway.Common.Models.PHSA;
+    using HealthGateway.Common.Services;
+    using HealthGateway.CommonTests.Utils;
+    using Microsoft.Extensions.Logging;
+    using Moq;
+    using Refit;
+    using Xunit;
+
+    /// <summary>
+    /// BroadcastService's Unit Tests.
+    /// </summary>
+    public class BroadcastServiceTests
+    {
+        private const string CategoryName = "Test Category Name";
+        private const string UnexpectedErrorMessage = "An unexpected error occurred while processing external call";
+        private const string ThrownExceptionMessage = "Error with HTTP Request";
+
+        /// <summary>
+        /// CreateBroadcast.
+        /// </summary>
+        [Fact]
+        public void ShouldCreateBroadcast()
+        {
+            // Arrange
+            IBroadcastService service = GetBroadcastService(null, HttpStatusCode.OK, false);
+
+            // Act
+            RequestResult<Broadcast> actualResult = service.CreateBroadcastAsync(new Broadcast()).Result;
+
+            // Assert
+            Assert.Equal(ResultType.Success, actualResult.ResultStatus);
+            Assert.NotNull(actualResult.ResourcePayload);
+            Assert.True(actualResult.TotalResultCount == 1);
+            Assert.Equal(CategoryName, actualResult.ResourcePayload.CategoryName);
+        }
+
+        /// <summary>
+        /// CreateBroadcast returns error.
+        /// </summary>
+        [Fact]
+        public void ShouldCreateBroadcastReturnsError()
```

Comment:
Naming seems odd for this and some of the other tests.  What about `CreateBroadcastShouldReturnError`?

Reply:
<img width="1218" alt="Screenshot 2022-10-27 at 11 38 54 AM" src="https://user-images.githubusercontent.com/58790456/198372130-7ca4c25b-bbd9-407d-9d82-e5bf225d1b14.png">

Reply:
Fixed grammar.

<img width="1264" alt="Screenshot 2022-10-27 at 11 47 52 AM" src="https://user-images.githubusercontent.com/58790456/198373834-df7d2597-9407-4677-a98f-03a8f40b0208.png">